packages/themes/pennwell/styles/components/_ads.scss

```diff
@@ -62,3 +62,53 @@
     box-shadow: 0 4px 8px 0 rgba(0, 0, 0, .2), 0 6px 20px 0 rgba(0, 0, 0, .19);
   }
 }
+
+.sticky-leaderboard {
+  &__container {
+    position: fixed;
+    right: 15px;
+    bottom: 0;
+    left: 15px;
+    z-index: 100000;
+    padding: map-get($spacers, 3);
+    text-align: center;
+    visibility: hidden;
```

Comment:
When hidden, does the container block the mouse from clicking on elements underneath it?

Reply:
No